### PR TITLE
add missing test for target being a primitive

### PR DESCRIPTION
**Pull request by stevemao**

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -100,6 +100,13 @@ it('should preserve property order', function () {
 	assert.equal(Object.keys(target).join(''), letters);
 });
 
+it('target can be primatives', function () {
```
Comment (SamVerschueren) [inline test.js]: Although I'd rather see AVA as testing framework, if `mocha` is used, the convention of titles is more like this:

```
it('should accept primitives as target', function () {
```

File: test.js [hunk 2]
```diff
@@ -100,6 +100,13 @@ it('should preserve property order', function () {
 	assert.equal(Object.keys(target).join(''), letters);
 });
 
+it('should accept primitives as target', function () {
+	var target = objectAssign('abcdefg', {foo: 'bar'});
+	var strObj = Object('abcdefg');
```
Comment (shinnn) [inline test.js]: `new Object('abcdefg')` looks better for me.

File: test.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ var assert = require('assert');
 Object.assign = undefined;
 var objectAssign = require('./');
 
-it('should have the correct length', function () {
+it('should expect at least 2 arguments', function () {
```
Comment (sindresorhus) [inline test.js]: This is incorrect. It's testing how many arguments are there in the code, just what's passed.

Comment (sindresorhus) [inline test.js]: Also, don't do unrelated changes when doing a PR ;)

Comment (stevemao) [inline test.js]: Hmm... from [mdn](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Function/length), it should mean the number of arguments expected by the function.

**Comment by stevemao** [comment]
I removed unrelated changes

File: test.js (hunk 3 above)
Comment (stevemao) [inline test.js]: `arguments.length` is how many arguments.

**Comment by sindresorhus** [comment]
Thanks :)
